projects/centre_console/inc/fault_monitor.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "status.h"
+#include "stdbool.h"
+#include "stdint.h"
+#include "watchdog.h"
+
+#define FAULT_MONITOR_WATCHDOG_TIMEOUT 4000
+
+typedef enum {
+  FAULT_STATUS_FAULT = 0,
+  FAULT_STATUS_ACK_FAULT,
+  FAULT_STATUS_OK,
+  NUM_FAULT_STATUS
+} FaultStatus;
+
+FaultStatus *get_fault_status(void);
```

Comment:
I understand why you did this but I don't think this is a good idea ... One option would be to create a test-only setter method.